docs/interface_specification.md

```diff
@@ -160,12 +160,16 @@ bp_cache_req_s, which contains the following fields.
   - Store Miss
   - Uncached Load
   - Uncached Store
-  - Writethrough Store
+  - Cache flush
+  - Cache clear
+  - Atomic
 - Physical address
 - Size (1B-64B)
 - Data (For uncached stores or writethroughs)
+- Hit
+- Subop type (amoswap, amolr, amosc, amoadd, amoxor, amoand, amoor, amomin, amomax, amominu, amomaxu)
 
-Additionally, the Cache Engine may require some metadata in order to service cache misses. This metadata may not be available at the same time as the request, due to the nature of high performance caches. The handshake here is valid-only. If a Cache Engines needs metadata in order to service the miss, it needs to be ready to accept metadata at any cycle later than or equal to the original cache miss. The cache is required to eventually provide this data, although not in any particular cycle. The current metadata fields are:
+Additionally, the Cache Engine may require some metadata in order to service cache misses. This metadata may not be available at the same time as the request, due to the nature of high performance caches. The handshake here is valid-only. If a Cache Engine needs metadata in order to service the miss, it needs to be ready to accept metadata at any cycle later than or equal to the original cache miss. The cache is required to provide this data within one cycle of the request. The current metadata fields are:
```

Comment:
Keep old sentence the same. Add "The latency between a cache request and its metadata must be a known constant for all message types, under all backpressure conditions.